.github/workflows/common_hive_reports.yaml

```diff
@@ -52,6 +47,16 @@ jobs:
         with:
           ref: main
 
+      - name: Build image
+        if: ${{ inputs.job_type != 'main' }}
+        run: make build-image
```

Comment:
This is a hacky way of approaching the issue. Can't you add a new `input` that contains the docker image? Adding `if` statements makes the code more complex